.github/workflows/060_generate_benchmark_weights.yml

```diff
@@ -0,0 +1,73 @@
+name: Generate benchmark weights
+
+on:
+  workflow_dispatch:
+
+jobs:
+  benchmark-weights:
+    runs-on: [self-hosted, tfchainrunner01]
+    container:
+      image: threefolddev/tfchain:4
+      env:
+        DEBIAN_FRONTEND: noninteractive
+        PATH: /root/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin
+        RUSTUP_HOME: /root/.rustup
+        CARGO_HOME: /root/.cargo
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Chown user
+        run: |
+          chown -R $USER:$USER $GITHUB_WORKSPACE
+
+      - name: Cache build
+        uses: actions/cache@v3
+        timeout-minutes: 6
+        continue-on-error: true
+        with:
+          path: |
+            /root/.cargo/bin/
+            /root/.cargo/registry/index/
+            /root/.cargo/registry/cache/
+            /root/.cargo/git/db/
+            substrate-node/target/
+          key: ${{ runner.os }}-tfchain-build-cache-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: ${{ runner.os }}-tfchain-build-cache
+
+      - name: Build
```

Comment:
I recall reading somewhere that the production profile is expected to generate more accurate numbers than the release profile. Would you double check this?

Reply:
It makes sense
I ll check

Reply:
Indeed
See here
https://docs.substrate.io/test/benchmark/#adding-benchmarks
I will change it

Reply:
After defining a `production` profile for compilation (equals to `release` profile, same as defined [here](https://github.com/paritytech/substrate/blob/033d4e86cc7eff0066cd376b9375f815761d653c/Cargo.toml#L382)) and testing it gave some errors compiling some lib.
I decided to opt for the [`bench` built-in profile](https://doc.rust-lang.org/cargo/reference/profiles.html#bench) which seems also appropriated.

Reply:
> What is the errors?

Not able to reproduce after rebase.
Seems to be ok now with polkadot upgrading.

So I switched and added production profile same as substrate https://github.com/paritytech/substrate/blob/033d4e86cc7eff0066cd376b9375f815761d653c/Cargo.toml#L382